### PR TITLE
Add contained-folder-count and contained-file-count webdav props

### PR DESCRIPTION
**Pull request by eneiluj**
requires https://github.com/nextcloud/server/pull/31193

Is there any other place where this should be mentioned?

### CODE REVIEW

**Review by CarlSchwan — APPROVED** [review]

**Review by tcitworld — APPROVED** [review]

File: developer_manual/client_apis/WebDAV/basic.rst [hunk 1]
```diff
@@ -96,6 +98,8 @@ The following properties are supported:
 - :code:`{http://nextcloud.org/ns}has-preview`
 - :code:`{http://owncloud.org/ns}size` Unlike :code:`getcontentlength`, this property also works for folders reporting the size of everything in the folder.
 - :code:`{http://nextcloud.org/ns}rich-workspace` this property is provided by the text app
+- :code:`{http://nextcloud.org/ns}contained-folder-count` The number of folders contained in the folder
```
Comment (juliushaertl) [inline developer_manual/client_apis/WebDAV/basic.rst:101]: Maybe additionally clarify that the counting is only on the first depth level, not recursively counting files/folders.

Comment (eneiluj) [reply developer_manual/client_apis/WebDAV/basic.rst]: :+1:

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**